sheepdog/utils/__init__.py

```diff
@@ -41,6 +42,7 @@
     submitted_state(),
     UPLOADING_STATE,
     UPLOADED_STATE,
+    DELETE_STATE
```

Comment:
Is this a desired behavior? i.e. a file node has been deleted, but we still allow file upload?

Reply:
Yes, it's basically a reupload.